pkg/cache/internal/informers.go

```diff
@@ -36,6 +36,7 @@ import (
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/cache"
```

Comment:
I think unit tests on some level would be good to get coverage for the new functionality

Reply:
Added unit tests